examples/hello_world/client/android/java/com/google/oak/hello_world/res/layout/activity_main.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:padding="16dp"
+    android:orientation="vertical"
+    android:gravity="center_vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/helloTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:textSize="32dp"
+        android:textColor="@color/green"
+        android:text="@string/empty"/>
+
+    <EditText
+        android:id="@+id/ipInput"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/ip_address">
+    </EditText>
+
+    <EditText
+        android:id="@+id/tcpInput"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/tcp_port">
+    </EditText>
```

Comment:
Not a big deal, but how about just having a single text field for address:port? That's all you are doing anyways with these two text fields it seems. Anyways up to you, the current version is fine.